WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -625,7 +625,8 @@ final class EditableOrderViewModel: ObservableObject {
     /// - Parameter item: Item to remove from the order
     ///
     func removeItemFromOrder(_ item: OrderItem) {
-        guard let input = createUpdateProductInput(item: item, quantity: 0) else { return }
+        guard let input = createUpdateProductInput(item: item,
+                                                   quantity: 0) else { return }
```

Comment:
nit: reverting the diffs on `createUpdateProductInput` usage in this file would make the PR and git history neater